src/backoffice/settings/dashboards/health-check/views/health-check-group.element.ts

```diff
@@ -0,0 +1,204 @@
+import { UUIButtonState } from '@umbraco-ui/uui';
+import { UUITextStyles } from '@umbraco-ui/uui-css/lib';
+import { css, html, nothing } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+
+import { UmbHealthCheckContext } from '../health-check.context';
+import { UmbHealthCheckDashboardContext } from '../health-check-dashboard.context';
+import {
+	HealthCheckAction,
+	HealthCheckGroupWithResult,
+	HealthCheckResource,
+	HealthCheckWithResult,
+	StatusResultType,
+} from '@umbraco-cms/backend-api';
+import { UmbLitElement } from '@umbraco-cms/element';
+import { tryExecuteAndNotify } from '@umbraco-cms/resources';
+import './health-check-action.element';
+
+@customElement('umb-dashboard-health-check-group')
+export class UmbDashboardHealthCheckGroupElement extends UmbLitElement {
+	static styles = [
+		UUITextStyles,
+		css`
+			uui-box {
+				margin-bottom: var(--uui-size-space-5);
+			}
+
+			p {
+				margin: 0;
+			}
+
+			.header {
+				display: flex;
+				justify-content: space-between;
+				align-items: center;
+			}
+
+			.check-results-wrapper {
+				margin-top: var(--uui-size-space-5);
+			}
+
+			.check-results-wrapper .check-result:not(:first-child) {
+				padding-top: var(--uui-size-space-5);
+				border-top: 1px solid var(--uui-color-divider-standalone);
+			}
+
+			.check-results-wrapper uui-button {
+				margin-block-start: 1em;
+			}
+
+			.check-result-description {
+				display: flex;
+			}
+
+			.check-result-description span {
+				width: 36px;
+			}
+
+			uui-icon {
+				vertical-align: sub;
+			}
+		`,
+	];
+
+	@property()
+	groupName!: string;
+
+	@state()
+	private _buttonState: UUIButtonState;
+
+	@state()
+	private _group?: HealthCheckGroupWithResult;
+
+	private _healthCheckContext?: UmbHealthCheckDashboardContext;
+
+	@state()
+	private _checks?: HealthCheckWithResult[] | null;
+
+	@state()
+	private _keyResults?: any;
+
+	private _api?: UmbHealthCheckContext;
+
+	constructor() {
+		super();
+		this.consumeContext<UmbHealthCheckDashboardContext>('umbHealthCheckDashboard', (instance) => {
```

Comment:
We need an `UmbContextToken` for this context, which it can export itself in the same file where the dashboard context is being created.

```ts
export const UMB_HEALTHCHECK_DASHBOARD_CONTEXT_TOKEN = new UmbContextToken<UmbHealthCheckDashboardContext>('umbHealthCheckDashboard');
```